l10n_es_aeat_mod190/tests/test_mod190.py

```diff
@@ -156,14 +158,15 @@ def test_mod190_multikeys(self):
             lambda r: r.partner_id == self.supplier
         )
         self.assertTrue(supplier_record)
-        self.assertEqual(2, len(supplier_record))
+        self.assertEqual(1, len(supplier_record))
```

Comment:
Estos tests no deberían cambiar

Reply:
Hello @pedrobaeza 
It's found only one supplier record

Reply:
If no change in the test data, there shouldn't be changed in the tests results.

Reply:
The problem probably comes from the groups added in the fields. You should add twice the same field, one with the groups, and another invisible without.